package.json

```diff
@@ -1,156 +1,12 @@
 {
-  "name": "@webex/widgets",
-  "version": "1.27.4",
-  "engines": {
-    "node": ">=20.13.1"
-  },
-  "main": "dist/webexWidgets.esm.js",
-  "module": "dist/webexWidgets.esm.js",
-  "scripts": {
-    "build": "rollup -c",
-    "demo:serve": "webpack serve ./demo/App.jsx --mode development",
-    "demo:build": "webpack ./demo/App.jsx --mode production",
-    "release": "semantic-release",
-    "release:debug": "semantic-release --debug",
-    "release:dry-run": "semantic-release --dry-run",
-    "start": "npm run demo:serve",
-    "test:e2e": "npm run demo:build && wdio wdio.conf.js",
-    "test:eslint": "eslint src/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/webex/widgets.git"
-  },
-  "publishConfig": {
-    "tag": "latest",
-    "access": "public"
-  },
+  "name": "webex-widgets",
+  "version": "1.0.0",
```

Comment:
Version can be anything since its the main package and its not gonna be published.

Reply:
Removed the version for better :-)